browser/main/lib/AwsMobileAnalyticsConfig.js

```diff
@@ -2,14 +2,31 @@ const AWS = require('aws-sdk')
 const AMA = require('aws-sdk-mobile-analytics')
 const ConfigManager = require('browser/main/lib/ConfigManager')
 
+const remote = require('electron').remote
+const os = require('os')
+
 AWS.config.region = 'us-east-1'
 if (process.env.NODE_ENV === 'production' && ConfigManager.default.get().amaEnabled) {
   AWS.config.credentials = new AWS.CognitoIdentityCredentials({
     IdentityPoolId: 'us-east-1:xxxxxxxxxxxxxxxxxxxxxxxxx'
   })
+
+  const rawPlatform = os.platform()
+
+  let actualPlatform
+  if (rawPlatform === 'darwin') {
+    actualPlatform = 'MacOS'
+  } else if (rawPlatform === 'win32') {
+    actualPlatform = 'Windows'
+  } else if (rawPlatform === 'linux') {
+    actualPlatform = 'Linux'
+  }
```

Comment:
I'm not sure whether we should change the platform name because AMA.Manager allows any string (ref: https://aws.github.io/aws-sdk-mobile-analytics-js/doc/AMA.Client.html#Options).

Reply:
I think we should change the platform name because aws-sdk-mobile-analytics-js 's README says `Optional valid values: 'Android', 'iPhoneOS', 'WindowsPhone', 'Blackberry', 'Windows', 'MacOS', 'Linux'` . ( ref: https://github.com/aws/aws-sdk-mobile-analytics-js/blob/master/README.md/#add-app-details-to-events)

Reply:
Alright, I understand.

Reply:
So, I think it's a better way to create a function because it's not easy to get why it needs to be changed.

```js
function convertPlatformName (platformName) {
  if (rawPlatform === 'darwin') {
    return 'MacOS'
  } else if (rawPlatform === 'win32') {
    return 'Windows'
  } else if (rawPlatform === 'linux') {
    return 'Linux'
  } else {
    // 'Other' is better, but it does not support...
    return ''
  }
}

const validPlatformName = convertPlatformName(os.platform())
```